src/include/stir/listmode/LmToProjDataAbstract.h

```diff
@@ -46,7 +47,7 @@ class LmToProjDataAbstract : public ParsingObject
     virtual Succeeded set_up() { return Succeeded::yes; }
 
     //! This function does the actual work
-    virtual void process_data() = 0;
+    virtual void process_data(shared_ptr<ProjData> proj_data_sptr = nullptr) = 0;
```

Comment:
not sure why you'd not put it in the base class?

Reply:
Just because the other two child classes don't seem to implement process_data or create outputs, so it doesn't seem to be a generally required feature.

Reply:
ok. that base class doesn't seem to serve any function at the moment. It was introduced to accommodate for the NiftyPET listmode processing, but isn't really used. To make it useful would require a lot of extra work, so let's not go there.

Conclusion: add it to `LmToProjData` only.

Reply:
Perfect, will do!